This patch is for speech-tools for Debian GNU/Linux,
so as to use the standard library rather than its older form.

Index: b/include/EST_dynamic_model.h
===================================================================
--- a/include/EST_dynamic_model.h
+++ b/include/EST_dynamic_model.h
@@ -39,7 +39,7 @@
 
 #include <cstdlib>
 #include <stdio.h>
-#include <fstream.h>
+#include <fstream>
 #include "EST.h"
 #include "EST_model_types.h"
 
